src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -460,8 +469,8 @@ def app_get_build_log(cmd, client, resource_group, service, name, deployment=Non
         raise CLIError(NO_PRODUCTION_DEPLOYMENT_ERROR)
     deployment_properties = client.deployments.get(
         resource_group, service, name, deployment).properties
-    if deployment_properties.source.type == "Jar":
-        raise CLIError("Jar deployment has no build logs.")
+    if deployment_properties.source.type == "Jar" or deployment_properties.source.type == "NetCoreZip":
```

Comment:
NIT: Use enum comparasion may avoid some string change in the future.

Reply:
That's the legacy string, we can have another PR to do refactor in a later PR.